diff --git a/node_modules/@pattern-lab/engine-twig/lib/engine_twig.js b/node_modules/@pattern-lab/engine-twig/lib/engine_twig.js
index 10acdbf..ea62abb 100644
--- a/node_modules/@pattern-lab/engine-twig/lib/engine_twig.js
+++ b/node_modules/@pattern-lab/engine-twig/lib/engine_twig.js
@@ -130,7 +130,7 @@ var engine_twig = {
 
   // render it
   renderPattern: function renderPattern(pattern, data, partials) {
-    var patternPath = pattern.relPath;
+    var patternPath = pattern.basePattern ? pattern.basePattern.relPath : pattern.relPath;
     if (patternPath.lastIndexOf(metaPath) === 0) {
       patternPath = patternPath.substring(metaPath.length + 1);
     }
